foodsaving/groups/models.py

```diff
@@ -37,6 +38,12 @@ class Group(BaseModel, LocationModel, ConversationMixin):
         on_delete=models.SET_NULL
     )
 
+    def active_member_count(self):
```

Comment:
maybe should be `approved_member_count`? It's a different concept than being `active`.